apps/passport-server/migrations/27_telegram_bot.sql

```diff
@@ -0,0 +1,10 @@
+-- Users that are engaged in a conversation
+create table telegram_bot_conversations (
+  id SERIAL PRIMARY KEY,
+  -- user_id corresponding the Telegram user
+  telegram_user_id BIGINT NOT NULL UNIQUE,
+  -- chat_id between the bot and the Telegram user
+  telegram_chat_id BIGINT NOT NULL UNIQUE,
```

Comment:
i do wonder whether these uniqueness constraints hold - what if a user deletes a conversation and then starts another one